code/modules/hydroponics/grown/root.dm

```diff
@@ -81,20 +83,21 @@
 
 /obj/item/reagent_containers/food/snacks/grown/whitebeet
 	seed = /obj/item/seeds/whitebeet
-	name = "white-beet"
-	desc = "You can't beat white-beet."
+	name = "white beet"
+	desc = "You can't beat white beet."
 	icon_state = "whitebeet"
 	filling_color = "#F4A460"
 	bitesize_mod = 2
+	tastes = list("white beet" = 1)
 	wine_power = 0.4
 
 // Red Beet
 /obj/item/seeds/redbeet
 	name = "pack of redbeet seeds"
 	desc = "These seeds grow into red beet producing plants."
-	icon_state = "seed-redbeet"
+	icon_state = "seedredbeet"
```

Comment:
was this invisible before? If not, then you'll have to change the name in the icons file or this will break it.

Reply:
that was an oopsie. I changed it back.